app/services/csv_import/litigation_sides.rb

```diff
@@ -24,7 +24,7 @@ def resource_klass
 
     def prepare_litigation_side(row)
       find_record_by(:id, row) ||
-        LitigationSide.find_or_initialize_by(
+        LitigationSide.new(
```

Comment:
If not find_or_initalize then `resource_klass.new` because assign attributes from `row` is already done later.

Reply:
But as I wrote, maybe combination of `litigation_id`, `name`, `side_type` would be good to prevent accidental duplicates.

Reply:
Or actually unique index on those values could be more beneficial. Thoughts?

Reply:
with a unique index it will stop duplicate rows from being added right? I think that's a good solution. So we assume that editing wouldn't happen in bulk like this but just additions, which I think is fair to say. Even to edit names of the sides, it probably makes more sense through the interface. Right?

Reply:
I think so. Editing is tricky as it's hard to determine what should always stay the same in the csv file.

Reply:
You still can edit it through bulk feature, after exporting existing, those will have litigation side `id` attached and the corresponding record will be found.